skbio/tree/tests/test_tree.py

```diff
@@ -303,6 +303,13 @@ def test_remove_node(self):
         self.assertEqual(len(n.children), 2)
         self.assertNotIn(n, self.simple_t.children)
 
+    def test_shear_prune_parent_dropped(self):
```

Comment:
Can you please add a test for `prune` specifically, in case `shear` doesn't use `prune` internally in the future?